knowledge_repo/app/templates/markdown-rendered.html

```diff
@@ -50,7 +50,7 @@
                   <div class="row">
                     <div class="col-md-11">
                       <div class="post_comment">
-                        Posted by <b>{{ item.author }}</b> on {{ item.created_at_fmt }}
+                        Posted by <b>{{ item.author }}</b> on {{ item.format_created_at }}
```

Comment:
This definitely works, but I wonder if we should instead add a jinja filter and use that instead? Something like http://stackoverflow.com/questions/4830535/python-how-do-i-format-a-date-in-jinja2 .